2.0/contrib/src/.bashrc

```diff
@@ -0,0 +1,2 @@
+# This will make scl collection binaries work out of box.
+source scl_source enable ruby200 ror40
```

Comment:
well yes but the .bashrc is read from the $HOME dir, which is /opt/openshift/src